### PR TITLE
**Breaking:** Update Paginator to new design.

### PR DESCRIPTION
**Pull request by ImogenF**
# Summary

New paginator design, as per the specs: 

https://app.zeplin.io/project/5cc31fad6f17562d6dac9a53/screen/5d08caa87bb35419d613efcd

# Jira ticket

https://contiamo.atlassian.net/secure/RapidBoard.jspa?rapidView=43&projectKey=UI&modal=detail&selectedIssue=UI-76&quickFilter=56

# Related issue

# To be tested

Me
- [ ] No error or warning in the console on `localhost:6060`

Tester 1

- [ ] Things look good on the demo.
  

Tester 2

- [ ] Things look good on the demo.

### CODE REVIEW

**Comment by TejasQ** [comment]
I changed it to Breaking because it "breaks" what is considered the old design.

File: src/Paginator/Paginator.tsx [hunk 1]
```diff
@@ -40,16 +39,18 @@ interface ControlProps {
   type: "first" | "previous" | "next" | "last"
 }
 
-const NavigationButton = styled(Button)<Partial<ControlProps>>(({ type }) => ({
-  width: type === "next" || type === "previous" ? 70 : 56,
-  marginRight: 3,
+const NavigationButton = styled(Button)(({ type, theme }) => ({
+  width: 96,
+  height: 24,
+  lineHeight: "24px",
```
Comment (TejasQ) [inline src/Paginator/Paginator.tsx]: It's a little odd that height just above is a number and this is a string even though it could also be a number. 🤔

**Comment by TejasQ** [comment]
Oops! We forgot to update the snapshots!

**Comment by TejasQ** [comment]
Maybe delete the snapshots. They always slow us down for no real benefit.

File: src/Paginator/Paginator.tsx (hunk 1 above)
Comment (ImogenF) [reply src/Paginator/Paginator.tsx]: The default unit for `lineHeight` is not `px`, it's just a multiplier of the font size. So `lineHeight: 24` is very different to `lineHeight: "24px"`

**Comment by fabien0102** [comment]
@ImogenF You have some conficts, can you rebase? 😃

**Pull request merged**